src/services/bindService.js

```diff
@@ -110,13 +110,20 @@ angular.module("openshiftCommonServices")
     };
 
     var isServiceBindable = function(serviceInstance, serviceClasses) {
-      var serviceClass = getServiceClassForInstance(serviceInstance, serviceClasses);
-
       // If being deleted, it is not bindable
       if (_.get(serviceInstance, 'metadata.deletionTimestamp')) {
         return false;
       }
 
+      // If not ready or failed, the service is not bindable
```

Comment:
We should be able to create a binding before the service is ready. It just won't complete until ready.

I'd change this check to only test the Failed condition.

Reply:
Ah yes, fixed. Thanks.